src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -80,6 +80,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
             Return SyntaxFactory.ParseLeadingTrivia(text)
         End Function
 
+        Public Function EscapeIdentifier(identifier As String) As String Implements ISyntaxFacts.EscapeIdentifier
+            Dim keywordKind = SyntaxFacts.GetKeywordKind(identifier)
```

Comment:
why is this not doing the same 0 check as C#?

Reply:
Came from http://sourceroslyn.io/#Microsoft.CodeAnalysis.VisualBasic.Workspaces/StringExtensions.vb,34 

That one doesn't check for null escape... so... /shrug

Willing to remove from the csharp impl